packages/core/src/Stores/client-store.js

```diff
@@ -1633,6 +1639,9 @@ export default class ClientStore extends BaseStore {
             this.resetMt5AccountListPopulation();
         }
         this.responseWebsiteStatus(await WS.wait('website_status'));
+        if (this.is_tnc_status_updated?.landing_company_short === 0) {
```

Comment:
could you please recheck `if this.is_tnc_status_updated?.landing_company_short === 0` is working as expected?

Reply:
Okay 🙏

Reply:
I'll check as soon as the mock api is ready